MovingBlocks/Terasology
# feature(autoconfig): migrate SystemConfig

### Contains

Migration SystemConfig to AutoConfig

Must merge with (https://github.com/Terasology/ModuleTestingEnvironment/pull/28)

### How to test
1. start game
2. change system settings
3. restart game(with exit from game)
4. Should works fine, without warnings about configs and settings must changed after restart.
